api/v1/callhomeconfig_types.go

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright 2021.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// CallHomeConfigSpec defines the desired state of CallHomeConfig
+type CallHomeConfigSpec struct {
+	ServiceAccountName      string `json:"serviceAccountName"`      //
```

Comment:
Aren't the service account credentials already included in the kubeconfig that is referenced below this?

Reply:
The kubeconfig refers to the one used to connect to the workload cluster. The service account name is the SA in the management cluster. Hope that explains it, if not, say it and I'll try to provide more details. But the first comment of the PR tries to describe that. LMK if it's still not clear.